superdesk/publish/transmitters/email.py

```diff
@@ -49,16 +56,70 @@ def _transmit(self, queue_item, subscriber):
             text_body = item.get('message_text', queue_item['formatted_item'])
             html_body = item.get('message_html', queue_item['formatted_item'])
 
+            # Attach feature media if required
+            attachments = []
+            if config.get('attach_media') and item.get('renditions'):
+                # The CID can be used to embbed the image in the html template email like <img src=CID:cid-value>
+                cid = config.get('media_cid', '')
+                # Get the rendition that has been nominated for attaching to the email
+                rendition = config.get('media_rendition', '')
+                media_item = item.get('renditions', {}).get(rendition)
+                if media_item and rendition:
+                    media = app.media.get(media_item['media'], resource='upload')
+                    im = Image.open(media)
+                    if config.get('watermark', False):
+                        im = self._get_watermark(im)
+
+                    binary = io.BytesIO()
+                    im.save(binary, 'jpeg', quality=80)
+
+                    attachments.append(Attachment(filename=media.name, content_type=media.content_type,
+                                                  data=binary.getvalue(),
+                                                  headers=[('Content-ID', cid)]))
+
             # sending email synchronously
             send_email(subject=subject,
                        sender=admins[0],
                        recipients=recipients,
                        text_body=text_body,
                        html_body=html_body,
-                       bcc=bcc)
+                       bcc=bcc,
+                       attachments=attachments)
 
         except Exception as ex:
             raise PublishEmailError.emailError(ex, queue_item.get('destination'))
 
+    def _get_watermark(self, image):
```

Comment:
imo, `get_watermark` does not belong here and can be moved to `superdesk.media.media_operations` module.

Reply:
OK will do!